pkg/scheduler/eventhandlers.go

```diff
@@ -134,6 +134,8 @@ func (sched *Scheduler) deleteNodeFromCache(obj interface{}) {
 		return
 	}
 
+	sched.SchedulingQueue.MoveAllToActiveOrBackoffQueue(logger, framework.NodeDelete, node, nil, nil)
```

Comment:
Right, the event doesn't seem to be used anywhere.

In any case, we should add a test, preferably integration.

Reply:
I added an integration test in #127387 https://github.com/kubernetes/kubernetes/blob/8b484be1573edf18298318c3fc4efaa009ab106e/test/integration/scheduler/plugins/plugins_test.go#L2889

These two pr are a little similar, #127387  trigger unscheduledPod add/update//delete event, while this one triggers node/delete event.

Maybe the integration test in #127387 can be more general and we can reuse it here.

Reply:
Added a new test case in `TestCoreResourceEnqueue`.